src/main/webapp/assets/modules/contest-detail/details/challengeGeneralInfo.js

```diff
@@ -1,4 +1,4 @@
-techlooper.directive('challengeGeneralInfo', function () {
+techlooper.directive('challengeGeneralInfo', function (localStorageService) {
```

Comment:
'techlooper' is not defined.<br>'localStorageService' is defined but never used.